### ISSUE TITLE
I can't get data from this.$user.data in AudioPlayer.PlaybackNearlyFinished

### ISSUE THREAD

**Issue by alvarocardenasc** [jovotech/jovo-framework#1220]
## I'm submitting a...

- [x] Bug report
- [x] Feature request
- [x] Documentation issue or request
- [x] Other... Please describe:

## Expected Behavior

get data from this.$user.data in AudioPlayer.PlaybackNearlyFinished alexa request  to put the next song in ENQUEUE

## Current Behavior

this.$user.data is empty {} when I try to access it in the AudioPlayer.PlaybackNearlyFinished handler

### Error log
```
If you have an error log, please paste it here.
```

## Your Environment

* Jovo Framework version used: 4.1.0
* Operating System:10.15.5

**Comment by aswetlow** [comment]
Hey @alvarocardenasc 
I'm not sure I got it right. Could you provide the code?

How and where did you add data into `this.$user.data`?

**Comment by alvarocardenasc** [comment]
I add the data in a function called playAudio():


```
async START() {
           /////////some condition here...
            return await this.playAudio();
    }



[used async for some API call inside]
async playAudio() {
         ///await some API call
         /// save info to session
         this.$session.data.audioPlayer = {
            playList: playList,
            playIndex: playIndex,
            song: playList[playIndex].song,
            token: playList[playIndex].token
        }
        ///save session data to user data.
        this.$user.data.audioPlayer = this.$session.data.audioPlayer;
        ///console.log(this.$user.data)  <-shows the info.
        return this.$send(AudioPlayerPlayOutput, {
                message: 'Starting audio',
                playBehavior: 'REPLACE_ALL',
                audioItem: {
                    stream: {
                        url: this.$session.data.audioPlayer.song,
                        token: this.$session.data.audioPlayer.token
                    },
                },
            });
}


 @Global()
    @Handle(AlexaHandles.onAudioPlayer('AudioPlayer.PlaybackNearlyFinished'))
    playbackNearlyFinished() {
        if (this.$user.data.audioPlayer && this.$user.data.audioPlayer.playList) {
            ///code...
        }
    }



```
I can access the info of this.$user.data on any intent handler but not inside any AlexaHandles.onAudioPlayer handler
there is no this.$user.data or this.$session.data there.

**Comment by aswetlow** [comment]
Hey @alvarocardenasc 
Sorry for the late reply. Found a bug in our Alexa getUserId implementation. Fixed it and we'll publish the fix very soon.

**Comment by jankoenig** [comment]
A fix was released with version `4.2.1`. Thanks @aswetlow for fixing and @alvarocardenasc for flagging 🎉
